test/Conversion/TorchToMhlo/reduction.mlir

```diff
@@ -241,3 +241,64 @@ func.func @torch.aten.max(%arg0: !torch.vtensor<[?,?,?],f32>) -> !torch.vtensor<
   %0 = torch.aten.max %arg0 : !torch.vtensor<[?,?,?],f32> -> !torch.vtensor<[],f32>
   return %0 : !torch.vtensor<[],f32>
 }
+
```

Comment:
Can we also add an e2e test for ForbeniusNorm?

Reply:
+1, new ops should always include e2e tests: https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#testing

> DO use an end-to-end test if you are implementing a new op or extending the support for an existing op.

Reply:
aten.frobenius_norm_dim comes from tracing PyTorch function torch.norm(data, p="fro"). Do you have any idea to use tracing mode in e2e tests, without introducing major changes to the current code structure.

Reply:
Will `torch.norm` decompose into `aten.frobenius_norm_dim` directly through scripting?

Reply:
> Will `torch.norm` decompose into `aten.frobenius_norm_dim` directly through scripting?

There is no decomposition rule for `torch.norm`-like op. If we script 'torch.norm' function, many unnecessary nested regions and unhandled ops will be introduced.

Reply:
I think it's fine to add tracing mode to e2e tests. And the compile has arguments for it, see: https://github.com/llvm/torch-mlir/blob/main/python/torch_mlir/__init__.py#L138

Reply:
> I think it's fine to add tracing mode to e2e tests. And the compile has arguments for it, see: https://github.com/llvm/torch-mlir/blob/main/python/torch_mlir/__init__.py#L138

I know it. But it seems for now, our e2e test backend configs all compile test cases by script mode. The simplest solution is that we change the compile function in e2e test backends to use trace mode for all test cases. I just want to figure out if there is a way to run one test case in trace mode but not modify the backend configs much.

Reply:
You should be able to do `torch.ops.aten.frobenius_norm(input_tensor, dim=..., keepdim=...)`. See: https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md#forward-method

Reply:
Thanks @ramiro050 . Two e2e test cases for AtenFrobeniusNormDim op are added.